lib/ledger_sync/ledgers/netsuite/resources/account.rb

```diff
@@ -6,9 +6,61 @@ module LedgerSync
   module Ledgers
     module NetSuite
       class Account < NetSuite::Resource
+
+         TYPES = {
+          'bank' => 'Bank',
+          'other_current_assets' => 'OthCurrAsset',
+          'fixed_asset' => 'FixedAsset',
+          'other_asset' => 'OthAsset',
+          'accounts_receivable' => 'AcctRec',
+          'equity' => 'Equity',
+          'expense' => 'Expense',
+          'other_expense' => 'OthExpense',
+          'cost_of_goods_sold' => 'COGS',
+          'accounts_payable' => 'AcctPay',
+          'credit_card' => 'CredCard',
+          'long_term_liability' => 'LongTermLiab',
+          'other_current_liability' => 'OthCurrLiab',
+          'income' => 'Income',
+          'other_income' => 'OthIncome',
+          'deferred_expense' => 'DeferExpense',
+          'unbilled_recievable' => 'UnbilledRec',
+          'deferred_revenue' => 'DeferRevenue',
+          'non_posting' => 'NonPosting'
+        }.freeze
+
+        TYPES_TO_CLASSIFICATION_MAPPING = {
+          'bank' => 'asset',
+          'other_current_assets' => 'asset',
+          'fixed_asset' => 'asset',
+          'other_asset' => 'asset',
+          'accounts_receivable' => 'asset',
+          'equity' => 'equity',
+          'expense' => 'expense',
+          'other_expense' => 'expense',
+          'cost_of_goods_sold' => 'expense',
+          'accounts_payable' => 'liability',
+          'credit_card' => 'liability',
+          'long_term_liability' => 'liability',
+          'other_current_liability' => 'liability',
+          'income' => 'revenue',
+          'other_income' => 'revenue',
+          'deferred_expense' => 'asset',
+          'unbilled_recievable' => 'assets',
+          'deferred_revenue' => 'liability'
+        }.freeze
```

Comment:
@ryanwjackson I copied this from the QBO account resource but it's not being used there either. How would we use this to populate the account classification values?

Reply:
These are QBO specific, so they won't belong here.  

These are used in the resource attribute types (see `Type::StringFromSet`) and then again in the de/serializers for the respective resoruces.

Reply:
@SeanBolt after some digging, I looked through the logs and found this: https://github.com/LedgerSync/ledger_sync/blob/master/lib/ledger_sync/ledgers/quickbooks_online/deserializer.rb#L16

It looks like a typo `TYPES` vs. `TYPE`.  I'll let you fix that here.  But I also believe this should be moved to the deserializer anyway.  I believe it can handle this logic in a block or a type.

Also, would be good to get a red/green test on this, because I'm guessing our request bodies always return a value and skip this block.